the_debuginator.h

```diff
@@ -2377,6 +2377,30 @@ void debuginator_move_to_parent(TheDebuginator* debuginator) {
 	}
 }
 
+void debuginator_move_to_root(TheDebuginator* debuginator) {
+	DebuginatorItem* hot_item = debuginator->hot_item;
+	while (hot_item && hot_item != debuginator->root) {
```

Comment:
Haha! Ok, so let's do it like this. I'm going to push my collapse feature first, then you can fix this PR so that it works fine with that. I thought I was almost done but then I realized load/save didn't work as I wanted so I've had to rewrite that too a bit. So it won't be until early next week probably.